### PR TITLE
Add MinecraftTurtle/MinecraftStuff commands to scratch2mcpi.

### PR DESCRIPTION
**Pull request by scratch2mcpiTurtle**
MinecraftTurtle install directory is scratch2mcpi/mcturtle.
MinecraftStuff install directory is scratch2mcpi/mcstuff. 

Add **init**.py empty file to scratch2mcpi/{mcturtle|mcstuff} directory.

See(Line30-40 and Line54-64):
https://github.com/naominix/naominix.github.io/blob/master/install.sh

### CODE REVIEW

**Comment by naominix** [comment]
- Additional Information

Remove unnecessary broadcast setTurtleX/Y/Z commands from scratch2mcpi.py.

File: scratch2mcpi.py [hunk 1]
```diff
@@ -44,6 +47,35 @@ def listen(s, mc):
...
+  turtleX = 0
+  turtleY = 0
+  turtleZ = 0
+  headingangle = 0
+  penBlockId = block.DIRT.id
+  penBlockData = 0
+  mc = minecraft.Minecraft.create()
+  pos = mc.player.getPos()
+  steve = turtle.MinecraftTurtle(mc,pos)
+  spos = steve.position
```
Comment (champierre) [inline scratch2mcpi.py]: Please replace "spos" with "stevePos".

File: scratch2mcpi.py [hunk 2]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
           if (is_number(mc, 'mcpiX', mcpiX) and is_number(mc, 'mcpiY', mcpiY) and is_number(mc, 'mcpiZ', mcpiZ) and is_number(mc, 'blockTypeId', blockTypeId) and is_number(mc, 'blockData', blockData)):
             mc.setBlock(mcpiX, mcpiY, mcpiZ, blockTypeId, blockData)
             print "setBlock: %d %d %d %d %d" % (mcpiX, mcpiY, mcpiZ, blockTypeId, blockData)
-        elif msg[1] == 'setBlocks':
+        # Minecraft Graphics Turtle(Start)
+	elif msg[1] == 'initTurtle':
+            steve = turtle.MinecraftTurtle(mc,spos)
```
Comment (champierre) [inline scratch2mcpi.py]: spos => stevePos

File: scratch2mcpi.py [hunk 3]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+	elif msg[1] == 'resetShapePoints':
+	    shapePoints = []
+            mcdrawing = stuff.MinecraftDrawing(mc)
+	elif msg[1] == 'setShapePoints':
+	    shapePoints.append(minecraft.Vec3(int(x1),int(y1),int(z1)))
+	    print "append.shapePoints:"
+	    print ' '.join(str(p) for p in shapePoints)
+	elif msg[1] == 'drawFace':
+	    if (fill == 'True'):
+		    fillFlag = True
```
Comment (champierre) [inline scratch2mcpi.py]: Indent is wrong.

File: scratch2mcpi.py [hunk 4]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+          if is_number(mc, 'headingangle', headingangle) :
+	    steve.setverticalheading(headingangle)
+	    print "steve.setverticalheading: (%d)" % (headingangle)
+        # Minecraft Graphics Turtle(End)
+        # Minecraft Stuff(Start)
+	elif msg[1] == 'drawLine':
+	    mcdrawing.drawLine(int(x1),int(y1),int(z1),int(turtleX),int(turtleY),int(turtleZ),blockTypeId, blockData)          		  
+	    print "mcdrawing.drawLine: (%d, %d, %d, %d, %d, %d, %d, %d)" % (x1,y1,z1,turtleX,turtleY,turtleZ,blockTypeId, blockData)
+	elif msg[1] == 'drawSphere':
+	    mcdrawing.drawSphere(turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)
```
Comment (champierre) [inline scratch2mcpi.py]: Please add space right after comma like

```
(turtleX, turtleY, turtleZ, ...
```

File: scratch2mcpi.py [hunk 5]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+        # Minecraft Graphics Turtle(End)
+        # Minecraft Stuff(Start)
+	elif msg[1] == 'drawLine':
+	    mcdrawing.drawLine(int(x1),int(y1),int(z1),int(turtleX),int(turtleY),int(turtleZ),blockTypeId, blockData)          		  
+	    print "mcdrawing.drawLine: (%d, %d, %d, %d, %d, %d, %d, %d)" % (x1,y1,z1,turtleX,turtleY,turtleZ,blockTypeId, blockData)
+	elif msg[1] == 'drawSphere':
+	    mcdrawing.drawSphere(turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)          		  
+	    print "mcdrawing.drawSphere: (%d, %d, %d, %d, %d, %d)" % (turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)
+	elif msg[1] == 'drawCircle':
+	    mcdrawing.drawCircle(turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)
```
Comment (champierre) [inline scratch2mcpi.py]: Please add space right after comma like

```
(turtleX, turtleY, turtleZ, ...
```

File: scratch2mcpi.py [hunk 6]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+	    steve.setverticalheading(headingangle)
+	    print "steve.setverticalheading: (%d)" % (headingangle)
+        # Minecraft Graphics Turtle(End)
+        # Minecraft Stuff(Start)
+	elif msg[1] == 'drawLine':
+	    mcdrawing.drawLine(int(x1),int(y1),int(z1),int(turtleX),int(turtleY),int(turtleZ),blockTypeId, blockData)          		  
+	    print "mcdrawing.drawLine: (%d, %d, %d, %d, %d, %d, %d, %d)" % (x1,y1,z1,turtleX,turtleY,turtleZ,blockTypeId, blockData)
+	elif msg[1] == 'drawSphere':
+	    mcdrawing.drawSphere(turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)          		  
+	    print "mcdrawing.drawSphere: (%d, %d, %d, %d, %d, %d)" % (turtleX,turtleY,turtleZ,radius,blockTypeId, blockData)
```
Comment (champierre) [inline scratch2mcpi.py]: Please add space right after comma like

```
(turtleX, turtleY, turtleZ, ...
```

File: scratch2mcpi.py [hunk 7]
```diff
@@ -114,6 +233,28 @@ def listen(s, mc):
...
+        backward = msg[1].get('backward', backward)
+        right = msg[1].get('right', right)
+        left = msg[1].get('left', left)
+        up = msg[1].get('up', up)
+        down = msg[1].get('down', down)
+        speed = msg[1].get('speed', speed)
+        turtleX = msg[1].get('turtleX', turtleX)
+        turtleY = msg[1].get('turtleY', turtleY)
+        turtleZ = msg[1].get('turtleZ', turtleZ)
+        headingangle = msg[1].get('headingangle', headingangle)
```
Comment (champierre) [inline scratch2mcpi.py]: headingangle => headingAngle

File: scratch2mcpi.py (hunk 2 above)
Comment (champierre) [inline scratch2mcpi.py]:
```
(mc,spos)
```

should be

```
(mc, spos)
```

File: scratch2mcpi.py [hunk 8]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+	    steve.penblock(penBlockId, penBlockData)          		  
+	    print "steve.penblock: (%s, %d)" % (penBlockId, penBlockData)
+	elif msg[1] == 'penup':
+	    steve.penup()
+	    print "steve.penup"
+	elif msg[1] == 'pendown':
+	    steve.pendown()
+	    print "steve.pendown"
+	elif msg[1] == 'setHeading':
+          if is_number(mc, 'headingangle', headingangle) :
```
Comment (champierre) [inline scratch2mcpi.py]: headingangle => headingAngle

File: scratch2mcpi.py [hunk 9]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+          if is_number(mc, 'speed', speed) :
+	    steve.speed(speed)          		  
+	    print "steve.speed: (%d)" % (speed)
+	elif msg[1] == 'setPenBlockId':
+	    steve.penblock(penBlockId)          		  
+	    print "steve.penblock: (%s)" % (penBlockId)
+	elif msg[1] == 'setPenBlockData':
+	    steve.penblock(penBlockId, penBlockData)          		  
+	    print "steve.penblock: (%s, %d)" % (penBlockId, penBlockData)
+	elif msg[1] == 'penup':
```
Comment (champierre) [inline scratch2mcpi.py]: penup => penUp

File: scratch2mcpi.py [hunk 10]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+	    steve.penup()
+	    print "steve.penup"
+	elif msg[1] == 'pendown':
```
Comment (champierre) [inline scratch2mcpi.py]: pendown => penDown

File: scratch2mcpi.py [hunk 11]
```diff
@@ -61,12 +93,99 @@ def listen(s, mc):
...
+		    fillFlag = True
+            elif (fill == 'False'):
+		    fillFlag = False
+	    mcdrawing.drawFace(shapePoints, fillFlag, blockTypeId)          		  
+	    #print "mcdrawing.drawFace: (%d, %d, %d)" % (shapePoints,fill,blockTypeId )
+	    print "mcdrawing.drawFace:"
+	    print ' '.join(str(p) for p in shapePoints)
+	    print(fill)
+	    print(blockTypeId)
+        # Minecraft Stuff(ENd)
```
Comment (champierre) [inline scratch2mcpi.py]: ENd => End

File: scratch2mcpi.py [hunk 12]
```diff
@@ -138,6 +279,34 @@ def main():
...
+      s.broadcast("setDown")
+      s.broadcast("setSpeed")
+      s.broadcast("setTurtleX")
+      s.broadcast("setTurtleY")
+      s.broadcast("setTurtleZ")
+      s.broadcast("setPosTurtle")
+      s.broadcast("initTurtle")
+      s.broadcast("setPenBlockId")
+      s.broadcast("setPenBlockData")
+      s.broadcast("penup")
```
Comment (champierre) [inline scratch2mcpi.py]: penup => penUp

File: scratch2mcpi.py [hunk 13]
```diff
@@ -138,6 +279,34 @@ def main():
...
+      s.broadcast("pendown")
```
Comment (champierre) [inline scratch2mcpi.py]: pendown => penDown

**Pull request merged**